code/game/objects/items/ego_weapons/subtype/wielded.dm

```diff
@@ -46,6 +46,8 @@
 
 //Destroy setup
 /obj/item/ego_weapon/wield/Destroy(mob/user)
+	if(!user)
```

Comment:
why does this accept the user anyway???
shouldnt it be using the `var/mob/current_holder` it has?
i dont think it even HAS anything calling it with the user var

Reply:
That's fair. I'll clean up the code.